packages/styled-icons/README.md

```diff
@@ -53,7 +56,36 @@ or
 npm install styled-icons --save
 ```
 
-Additionally, you will need to have installed a version of `styled-components` at least version 4.1.0 or newer, as `styled-icons` depends on `styled-components` as a peer dependency.
+Alternatively you can install only the icon packs you need:
+
+```
+yarn add @styled-icons/boxicons-logos
+yarn add @styled-icons/boxicons-regular
+yarn add @styled-icons/boxicons-solid
+yarn add @styled-icons/crypto
+yarn add @styled-icons/entypo
+yarn add @styled-icons/entypo-social
+yarn add @styled-icons/evil
+yarn add @styled-icons/fa-brands
+yarn add @styled-icons/fa-regular
+yarn add @styled-icons/fa-solid
+yarn add @styled-icons/feather
+yarn add @styled-icons/foundation
+yarn add @styled-icons/heroicons-outline
+yarn add @styled-icons/heroicons-solid
+yarn add @styled-icons/icomoon
+yarn add @styled-icons/material
+yarn add @styled-icons/material-outlined
+yarn add @styled-icons/material-rounded
+yarn add @styled-icons/material-twotone
+yarn add @styled-icons/material-sharp
+yarn add @styled-icons/octicons
+yarn add @styled-icons/open-iconic
+yarn add @styled-icons/typicons
+yarn add @styled-icons/zondicons
+```
+
+Finally, you will need to have installed a version of `styled-components` at least version 4.1.0 or newer, as `styled-icons` depends on `styled-components` as a peer dependency.
```

Comment:
I was thinking about this last night. I _think_ we could support other styled libraries as well by doing something like.

```js
import styled from '@emotion/styled'
StyledIconBase.styled = styled
```

This would allow implementors to choose which styled lib they want.

Reply:
Possibly yeah, though I'd worry about `emotion` or `styled-components` introducing something incompatible with the other library.  As in, supporting both would mean we could only use features common to both and might be limited in our ability to fully utilize one or the other.

There's also the issue that I'm not sure if the output of our build is compatible with `emotion` - the icons are pre-processed with the Styled Components babel plugin during build, and I'm not sure if the output of that is Emotion-compatible.

Reply:
I can dig into this at a later point and can easily be a non-breaking change.